src/mf/mf-hid.cpp

```diff
@@ -1,7 +1,7 @@
 // License: Apache 2.0. See LICENSE file in root directory.
 // Copyright(c) 2015 Intel Corporation. All Rights Reserved.
 
-#ifdef RS2_USE_WMF_BACKEND
+#ifdef WIN32
```

Comment:
Won't compile / run on Win7 I fear

Reply:
I'm planing to remove all the backend ifdefs and handle that on cmake